app/graph/types/query_type.rb

```diff
@@ -121,7 +121,10 @@ def find_public_team(slug:)
   end
 
   def project_media(ids:)
-    objid, tid = ids.split(",").map(&:to_i)
+    # Get the first and last id to handle the ids: in format '1,,2', '1,null,2' or '1,0,2'
```

Comment:
Question: Do we always get the ids with one element that might be empty, 0 or null in between them? Or does the quantity of elements vary, e.g.: '1,2,null,3'? Why do we always want the first and the last?

Reply:
The format for IDs is `ProjectMediaId,ProjectId,TeamId`, with both `ProjectId` and `TeamId` expected to be `nil`. Since `ProjectId` was removed but the external GraphQL still uses the old format, we now only need `ProjectMediaId` and `TeamId`. Therefore, the behavior changes to retrieving the first and last values to ensure we get `ProjectMediaId` and `TeamId` in case GraphQL submits three arguments.